File: Snankbar/app/src/main/java/com/example/snankbar/MainActivity.java
package com.example.snankbar;

import androidx.appcompat.app.AppCompatActivity;
import androidx.constraintlayout.widget.ConstraintLayout;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.google.android.material.snackbar.Snackbar;

public class MainActivity extends AppCompatActivity {
    private ConstraintLayout parent;
    private Button btn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        parent = findViewById(R.id.parent);

        btn = findViewById(R.id.button);

        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Snackbar.make(parent, "this is", Snackbar.LENGTH_INDEFINITE).show();
            }
        });

    }
}

File: LearnKotlin/app/src/main/java/com/example/learnkotlin/MainActivity.kt
package com.example.learnkotlin

import android.graphics.Color
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*
import kotlin.random.Random

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

       setNumber()
        btnLeft.setOnClickListener{
            var leftNum = btnLeft.text.toString().toInt();
            var rightNum = btnRight.text.toString().toInt();
            if(leftNum > rightNum)
            {
                Toast.makeText(this, "Correct", Toast.LENGTH_SHORT).show()
                theBackground.setBackgroundColor(Color.GREEN)
            }
            else {
                Toast.makeText(this, "Wrong", Toast.LENGTH_SHORT).show()
                theBackground.setBackgroundColor(Color.RED)
            }
            setNumber()
        }

        btnRight.setOnClickListener{
            var leftNum = btnLeft.text.toString().toInt();
            var rightNum = btnRight.text.toString().toInt();
            if(leftNum < rightNum)
            {
                Toast.makeText(this, "Correct", Toast.LENGTH_SHORT).show()
                theBackground.setBackgroundColor(Color.GREEN)
            }
            else {
                Toast.makeText(this, "Wrong", Toast.LENGTH_SHORT ).show()
                theBackground.setBackgroundColor(Color.RED)
            }
            setNumber()
        }
    }

    private fun setNumber() {
        var a = Random.nextInt(0,10);
        var b = Random.nextInt(0,10)
        while(a==b){
            b = Random.nextInt(0, 10)
        }
        btnLeft.text = a.toString();
        btnRight.text = b.toString();
        println(a)
    }
}